bait-files/src/test/java/com/google/sps/LoginServletTest.java

```diff
@@ -55,5 +55,6 @@ public void doPost_addsUserSuccessfully_returnsUserInDatabase() throws IOExcepti
 
         DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
         assertEquals(1, ds.prepare(new Query(USER_ENTITY)).countEntities(withLimit(10)));
+        verify(response, atLeast(1)).sendRedirect("/profile.jsp"); // only if you want to verify username was called...
```

Comment:
I don't understand the comment here.  The line of code seems like it speaks for itself: verify that a redirect to /profile.jsp happens.  I don't think a comment is necessary =)

Reply:
removed comment